h5c/vic/src/vic-webapp/src/styles.scss

```diff
@@ -15,3 +15,9 @@
 .cursor-pointer {
   cursor: pointer;
 }
+
+/* Fixes issue with clr-stack-block when some row bottom borders are not visible */
+.stack-view .stack-block .stack-block,
+.stack-view .stack-block {
```

Comment:
are both declarations needed? it seems like just '.stack-view .stack-block' would cover it.